tasks/pcs-configure-pcs-pcsd.yml

```diff
@@ -19,6 +19,33 @@
   when:
     - ha_cluster_regenerate_keys
 
+- name: Create certificate and private_key files
+  when: ha_cluster_pcsd_certificates | length > 0
+  block:
+    - name: Allow certmonger to write into pcsd's certificate directory
+      file:
```

Comment:
Hi @tomjelinek,
First, I created the cert and the key in the default location `/etc/pki/tls/{certs,private}` and moved the files to `/var/lib/pcsd` [0]. Then, I thought it'd be simpler for the certificate role to place them in `/var/lib/pcsd` directly. But we get this error `risky-file-permissions File permissions unset or incorrect` by the change. Without giving the permission to the certificate role, writing to the directory fails.

Should I go back to the original plan [0]? Or could there be a way to tell the github action to allow it? Thanks!

Reply:
You must set the required `mode` with the `file` module

Reply:
> You must set the required `mode` with the `file` module to fix this

Thank you, @spetrosi. Hmmm, my intention is assuming `/var/lib/pcsd` exists and change only the setype with other attributes unchanged using this feature...
```
If mode is not specified and the destination filesystem object does exist, the mode of the existing filesystem object will be used.
```
But if it solves the issue, let me play around it by getting the mode and resetting it.
Thanks!